dll/shellext/zipfldr/CZipExtract.cpp

```diff
@@ -344,6 +344,187 @@ class CZipExtract :
         PropertySheetW(&psh);
     }
 
+    eZipExtractError ExtractSingle(
+        HWND hDlg,
+        LPCSTR FullPath,
+        bool is_dir,
+        unz_file_info64 Info,
```

Comment:
Use a pointer or reference for the structure.

Reply:
Fixed.